AprilSylph/XKit-Rewritten
# Open in Tabs: Fix interaction with Tumblr Patio UI

### Description


Prevents Open in Tabs from intercepting a click in part of the "Tumblr Patio" feature preview UI that's used to add a blog column.

Resolves #1371.

### Testing steps

- Enable Open in Tabs.
- In Tumblr Patio, add a blog column and confirm that selecting the desired blog works correctly.
- Make sure that... it still... works as much as it did before otherwise? I dunno; I don't use the script.
